packages/infra/src/components/ecsWebAPI.ts

```diff
@@ -129,7 +131,10 @@ export class ECSWebAPI extends Construct {
         AWS_REGION: cdk.Stack.of(this).region,
 
         // Storage bucket name
-        S3_BUCKET_NAME: props.storageBucket.bucketName
+        S3_BUCKET_NAME: props.storageBucket.bucketName,
+
+        // Sentry DSN
+        ...(props.sentryDsn ? { SENTRY_DSN: props.sentryDsn } : {})
```

Comment:
JavaScript tangent, but wish there was a nicer way to write this.
I think this is fine if it's `string | undefined`
```javascript
  SENTRY_DSN: props.sentryDsn
```
Though technically there's a difference in that the property is present `'SENTRY_DSN' in object` true here, but false for your code. So if care about property presence than your code is more correct.

kinda wish you could do

```javascript
{
  foo?: <expression>
}
```